addons/velero/template/base/install.tmpl.sh

```diff
@@ -244,12 +244,33 @@ function velero_patch_args() {
     local src="$1"
     local dst="$2"
 
-    if [ "${VELERO_DISABLE_RESTIC}" = "1" ] || [ -z "${VELERO_RESTIC_TIMEOUT}" ]; then
-        return 0
+    # if the user has specified any additional velero server flags, add them to the velero deployment
+    if [ -n "$VELERO_SERVER_FLAGS" ]; then
+        # iterate over the flags in reverse order since they are prepended to the list of kustomize patches
+        IFS=',' read -ra flags <<< "$VELERO_SERVER_FLAGS"
```

Comment:
it may be safest to restore the value of IFS for any subsequent calls to the read utility

```suggestion
        oIFS="$IFS"; IFS=',' read -ra flags <<< "$VELERO_SERVER_FLAGS"; IFS="$oIFS"
```

Reply:
This seems ok as is based on https://unix.stackexchange.com/a/18889

Reply:
leaving this as-is since it would only be defined for the read invocation